### ISSUE TITLE
Consider `directives_ordering` for inclusion

### ISSUE THREAD

**Issue by jibiel** [dart-lang/lints#45]
Documentation: [`directives_ordering`](https://dart-lang.github.io/linter/lints/directives_ordering.html)

I find it very useful, especially in a team of a few people. Is there any particular reason not to include it?

I know this goes beyond Dart, but [`flutter_lints`](https://github.com/flutter/packages/tree/master/packages/flutter_lints) doesn't have a separate repo and doesn't accept issues.

**Comment by jibiel** [comment]
For some reason this doesn't throw any warnings locally but do throws them in CI.

```yaml
linter:
  rules:
    directives_ordering: true
```

Maybe it's a SDK thing.

Anyway, it won't allow me to split directives into additional sections:

```dart
import 'package:flutter/services.dart';
import 'package:path_provider/path_provider.dart';

import 'package:my_app/app.dart'; // info • Sort directive sections alphabetically • lib/app.dart:4:1 • directives_ordering
import 'package:my_app/models/index.dart';
```

so I will have to politely decline this rule.
